src/os/bluestore/BlueRocksEnv.cc

```diff
@@ -99,13 +99,13 @@ class BlueRocksRandomAccessFile : public rocksdb::RandomAccessFile {
 
   // Used by the file_reader_writer to decide if the ReadAhead wrapper
   // should simply forward the call and do not enact buffering or locking.
-  bool ShouldForwardRawRequest() const override {
```

Comment:
`BlueRocksRandomAccessFile` inherits from `rocksdb::RandomAccessFile`, and the latter has a virtual function
```c++
  virtual bool ShouldForwardRawRequest() const {
    return false;
  }
...
  virtual void EnableReadAhead() {}

```